src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesSoftDeletedVault.md

```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.dll-Help.xml
+Module Name: Az.RecoveryServices
+online version:
```

Comment:
Please fill in online version for new cmdlets. E.g. https://learn.microsoft.com/powershell/module/az.recoveryservices/get-azrecoveryservicessoftdeletedvault